src/messages.json

```diff
@@ -19,6 +19,21 @@
         "ActionsSimplificationModal.cancel": "Cancel",
         "ActionsSimplificationModal.save": "Save",
         "ActionsSimplificationModal.title": "Available Actions",
+        "AmusementPark.character": "the ticket",
+        "AmusementPark.entrance": "entrance",
+        "AmusementPark.ferrisWheel": "ferries wheel",
+        "AmusementPark.gameBooth": "game booth",
+        "AmusementPark.goKart": "go kart",
+        "AmusementPark.label": "An amusement park scene. A scene with your favourite rides; there's a roller coaster, ferris wheel, pirate ship, go kart, swing ride and merry go round! There's other fun things to do, like game booth and water park, and if you want some snack, there is a snack stand as well. Your character is a ticket.",
```

Comment:
I made a few small edits:

"An amusement park scene. A scene with your favourite rides; there's a roller coaster, ferris wheel, pirate ship, go karts, swing ride and merry go round! There's other fun things to do, like a game booth and water park, and if you want some snacks, there is a snack stand as well. Your character is a ticket."